ciao-controller/internal/datastore/datastore.go

```diff
@@ -1439,6 +1454,8 @@ func (ds *Datastore) GetTenantCNCISummary(cnci string) ([]types.TenantCNCI, erro
 
 	ds.tenantsLock.RUnlock()
 
+	sort.Sort(ByTenantID(cncis))
```

Comment:
ok - I put it there because I thought the api might want it sorted as well, however, probably that's better left to the client now that I think of it.